### PR TITLE
Create hello microverse

### PR DESCRIPTION
**Pull request by Ol-create**
- Add header with the text "hello microverse!" .
- Add style color to header class.
- test.md is not included.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.HDColor {
+  color:blueviolet;
+
+}
```
Comment (keroloussamy) [inline styles.css]: Kindly solve your error linter, you can solve it by 

1. Add space after `color:` like `color: blueviolet;`.
2. Delete line 3 the empty one.
3. Add an empty line at the end of the file.

Please don't submit until you are sure that your linter is passing.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @Ol-create, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (Ol-create) [reply styles.css]: Thank you.
I really appreciate your effort.

**Review by Donard97 — APPROVED** [review]
# Hi @Ol-create , ✋  

Good job implementing the required changes from the other reviewer. 👏 👏 💯 

**Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🎉🎉🎉** 

## Congratulations! 🎉  🎉  🎉 

giphy

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**